iroh-ctl/src/store.rs

```diff
@@ -0,0 +1,167 @@
+use std::path::PathBuf;
+
+use anyhow::Result;
+use cid::Cid;
+use clap::{Args, Subcommand};
+use iroh_rpc_client::Client;
+
+#[derive(Args, Debug, Clone)]
+pub struct Store {
+    #[clap(subcommand)]
+    command: StoreCommands,
+}
+
+#[derive(Subcommand, Debug, Clone)]
+pub enum StoreCommands {
+    #[clap(about = "Version of the iroh store binary")]
+    Version,
+    Block(Block),
+    Dag(Dag),
+    #[clap(hide = true)]
+    GetLinks {
+        cid: Cid,
+    },
+}
+
+#[derive(Args, Debug, Clone)]
+#[clap(about = "Interact with raw IPFS blocks")]
+pub struct Block {
+    #[clap(subcommand)]
+    command: BlockCommands,
+}
+
+#[derive(Subcommand, Debug, Clone)]
+pub enum BlockCommands {
+    #[clap(about = "Get a raw IPFS block.")]
+    Get { cid: Cid },
+    #[clap(
+        about = "Store input as an IPFS block.
+Not yet implemented.",
+        hide = true
+    )]
+    Put { path: PathBuf },
+    #[clap(
+        about = "Remove IPFS block(s).
+Not yet implemented.",
+        hide = true
+    )]
+    Rm {
+        cids: Vec<String>,
+        #[clap(short, long)]
+        force: bool,
+    },
+    #[clap(hide = true)]
+    Has { cid: Cid },
+}
+
+#[derive(Args, Debug, Clone)]
+#[clap(
+    about = "Interact with IPLD DAG objects.
+Not yet implemented.",
+    hide = true
+)]
+pub struct Dag {
+    #[clap(subcommand)]
+    command: DagCommands,
+}
+
+#[derive(Subcommand, Debug, Clone)]
+pub enum DagCommands {
+    #[clap(
+        about = "Streams the selected DAG as a .car stream on stdout.
```

Comment:
`car` can be default but should be configurable, as there are already two version of car files out there, and I suspect there might be others